teachopencadd/talktorials/T016_protein_ligand_interactions/talktorial.ipynb

```diff
@@ -4,201 +4,186 @@
    "cell_type": "markdown",
```

Comment:
<ol><li>Also halogen interactions with THR790, MET766</li><li>a small question: Does there need space between the residues(LYS) and the number(745)?</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/75/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>